bitnami/contour/Chart.yaml

```diff
@@ -12,7 +12,7 @@ sources:
   - https://github.com/projectcontour/contour
   - https://github.com/envoyproxy/envoy
   - https://github.com/bitnami/bitnami-docker-contour
-version: 1.0.1
+version: 1.0.2
```

Comment:
Please bump the mino version instead since you're including new features (parameters) in your PR